src/common/helper/documentVerificationService.ts

```
@@ -0,0 +1,73 @@
+import { InjectRepository } from '@nestjs/typeorm';
+// import { issuerDid } from '../../../src/config/issuerDid';
+// Adjust the path as per your project structure
+import { type verr } from '@sunbird-rc/verification-sdk'; // Hypothetical Verifieer SDK import
+import { UserDoc } from '@entities/user_docs.entity';
+import { promises as fs } from 'fs';
+// import path from 'path';
+import * as path from 'path';
+import { EncryptionService } from './encryptionService';
+
+async function verify(): Promise<{ verifyCredential: Function }> {
+  const module = await eval(`import('@sunbird-rc/verification-sdk')`);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)